build-faq.md

```diff
@@ -4,4 +4,58 @@ When building on macOS, the build system will make a native Cocoa app by default
 
 # Attempting to build the SDL frontend on macOS fails on linking
 
-SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
\ No newline at end of file
+SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
+
+# Windows build process
+
+For the various tools and libraries, follow the below guide to ensure easy, proper configuration for the build environment:
+
+#### clang
+
+This may be installed via a Visual Studio installer packages instead of built from source.
+
+#### SDL Port
+
+[libsdl2](https://libsdl.org/download-2.0.php) has two separate files that must be downloaded
+  1. The `-x86` Runtime Binary (e.g., `SDL2-2.0.12-win32-x86.zip` (as of writing))
+  2. The Visual C++ Development Library (e.g., `SDL2-devel-2.0.12-VC.zip` (as of writing))
+
+For the Runtime Binary, place the extracted `SDL2.dll` into a known folder for later.
+
+- `C:\SDL2\bin\SDL2.dll` will be used as an example
+
+For the Visual C++ Development Library, place the extracted files within a known folder for later.
+
+The following examples will be referenced later:
+
+- `C:\SDL2\lib\x86\*`
+- `C:\SDL2\include\*`
+
+#### Gnuwin
+
+Ensure that this is in %PATH%.
+
+If errors arise (i.e., particularly with the `CREATE_PROCESS('usr/bin/mkdir')` calls, also verify that Git for Windows has not been installed with full Linux support. If it has, remove `C:\Program Files\Git\usr\bin` from the SYSTEM %PATH% until after compilation.
+
+### Building
+
+Within a command prompt in the project directory:
+
+```
+vcvars32
+set path=%path%;C:\SDL2\bin
```

Comment:
On my Windows build VM I don't set `%path%`, and SDL2.dll (from the `-devel-` zip) is in the `lib/x86` folder. Can you confirm this works for you? It simplifies the process.

Reply:
This works for me as well. I made a mistake in that I thought I needed to add `SDL2.dll` to `%path%`, when I actually needed to add the `rgbds` files to path. I will remove the notes about the SDL2.dll (as well as downloading it separately, to address the other comment) and I will add a section specific to `rgdbgs` path adding.